src/Sylius/Bundle/ReviewBundle/Doctrine/ORM/Subscriber/LoadMetadataSubscriber.php

```diff
@@ -106,12 +104,12 @@ private function createReviewerMapping($reviewerEntity, ClassMetadata $reviewerE
         return [
             'fieldName' => 'author',
             'targetEntity' => $reviewerEntity,
-            'joinColumn' => [
+            'joinColumns' => [[
                 'name' => 'author_id',
                 'referencedColumnName' => $reviewerEntityMetadata->fieldMappings['id']['columnName'],
-                'nullable' => false,
+                'nullable' => true,
```

Comment:
Yes, review definitely should have an author in all cases, so it should not be nullable.